package jaleynet;

message Param {
	repeated NetParam net=1;
}


message NetParam{
	required string caffe_root=8;
	required string name=1;
	required string protopath=2;
	required string trainpath=10;
	required string solverpath=11;
	required string modelpath=3;
	optional bool has_mean=4[default=false];
	optional string meanpath=5;
	optional bool channel_swap=6[default=true];
	optional uint32 raw_scale=7;
	optional bool gpu=9[default=false];
}

